src/api/files.js

```diff
@@ -89,6 +89,14 @@ export function preview(file) {
 	return `${baseURL}/api/files/${file}?alt=media&preview`;
 }
 
+export function previewDocs(fileId) {
```

Comment:
This function should open the docs and not return the url. Also, its not preview so a better name will be something like "openEditOnline" or "editFileOnline".

Reply:
done